tips/tests/test_rule_engine.py

```diff
@@ -381,4 +362,22 @@ def test_list_assertion(self):
         rules = [
             {"type": "ref", "ref_id": "1"}
         ]        
-        self.assertTrue(apply_rules(test_data, rules, compound_rules))
\ No newline at end of file
+        self.assertTrue(apply_rules(test_data, rules, compound_rules))
+
+        user_data = objectpath.Tree({
+            "focus": [{
+                "typeBesluit": "Toekenning",
+                "soortProduct": "Minimafonds",
+                "processtappen": {
+                    "beslissing": {
+                        "datum": "2020-04-15T00:00:00+02:00"
+                    }
+                }
+            }]
+        })        
+        ret = user_data.execute("len($.focus.*[@.soortProduct is 'Minimafonds' and @.typeBesluit is 'Toekenning'])")
+        ret2 = user_data.execute("now() - timeDelta(1, 0, 0, 0, 0, 0)")
+        ret3 = user_data.execute("now() - timeDelta(1, 0, 0, 0, 0, 0) <= dateTime($.focus[0].processtappen.beslissing.datum)")
+        ret4 = user_data.execute("$.focus.*[@.soortProduct is 'Minimafonds' and @.typeBesluit is 'Toekenning' and now() - timeDelta(1, 0, 0, 0, 0, 0) <= dateTime(@.processtappen.beslissing.datum)]")
+        ret5 = user_data.execute("len($.focus.*[@.soortProduct is 'Minimafonds' and @.typeBesluit is 'Toekenning' and now() - timeDelta(1, 0, 0, 0, 0, 0) <= dateTime(@.processtappen.beslissing.datum)]) >= 1")
+        self.assertTrue(ret5)
```

Comment:
wat doe je met al die `ret`s? je doet er geen asserts op. of weg, of asserts voor maken.

Reply:
die mogen weg denk ik dan